src/main/java/bio/terra/validation/StudyRequestValidator.java

```diff
@@ -115,10 +115,24 @@ private void validateRelationship(RelationshipModel relationship, Errors errors,
         }
     }
 
-    private void validateAssetTable(AssetTableModel assetTable, Errors errors, SchemaValidationContext context) {
+    // returns true if the table is the root table
+    // error for invalid root column is determined within this method
+    private boolean validateAssetTable(
```

Comment:
nit: Can we update this name to something like `validateAssetTableAndCheckRoot` to distinguish between this and the other void validate methods?